cmd/mdmctl/apply_dep_profile.go

```diff
@@ -16,15 +18,23 @@ func (cmd *applyCommand) applyDEPProfile(args []string) error {
 	var (
 		flProfilePath = flagset.String("f", "", "filename of DEP profile to apply")
 		flTemplate    = flagset.Bool("template", false, "print a JSON example of a DEP profile")
+		flAnchorFile  = flagset.String("anchor", "", "filename of PEM certificate to add to trusted anchors")
```

Comment:
I can generate a self signed root CA, and then an intermediary, and then sign a cert with that intermediary. The whole chain needs to be added, no?